pkg/providers/aws/cluster_vpc.go

```diff
@@ -39,17 +39,17 @@ import (
 )
 
 const (
-	defaultSubnetPostfix          = "subnet-group"
-	defaultSecurityGroupPostfix   = "security-group"
-	defaultAWSPrivateSubnetTagKey = "kubernetes.io/role/internal-elb"
-	defaultSubnetGroupDesc        = "Subnet group created and managed by the Cloud Resource Operator"
+	// 28 has too few hosts available to be future-proof for RHOAM products, so use 27 to avoid a migration being
```

Comment:
@cecobask it's minor. Comments about masks order seems still not so clear. Maybe change it like following:
	// CIDR masks
	// In AWS this must be between 16 and 28
	// The larger the mask, the less hosts available in the network
	// We want to use the least host addresses possible, to support clusters provisioned in VPCs with small required in the future
    // 28 has too few hosts available to be future-proof for RHOAM products, so use 27 to avoid a migration being
I changed it a bit, could you please check, update if needed. Thank you

Reply:
Good spot. I didn't notice this happened when I sorted the lines alphabetically. I'll update it 👍🏻